Objects/setobject.c

```diff
@@ -631,8 +644,19 @@ set_merge(PySetObject *so, PyObject *otherset)
     return 0;
 }
 
+/*[clinic input]
+set.pop
+
+    so: setobject
```

Comment:
Personal style nit: can you omit the extra newline between the method def and the param list?

```suggestion
    so: setobject
```

Reply:
Updated. Btw, is there some styleguide for the AC definitions? The AC guide examples and lots of AC code does have the extra newline..

Reply:
> Updated. Btw, is there some styleguide for the AC definitions?

Thanks! No, there is no style guide. I has been asked about before, so perhaps we should add a recommendation to PEP-7.

> The AC guide examples and lots of AC code does have the extra newline..

I know; that is unfortunate. I'll update them. IMO, the slightly more compact form is more readable.